arch/arm/boot/dts/overlays/README

```diff
@@ -1762,6 +1762,21 @@ Params: oscillator              Clock frequency for the CAN controller (Hz)
 
         interrupt               GPIO for interrupt signal
 
+Name:   mcp25xxfd-spiX-Y
```

Comment:
Recent advances in the overlay parameter mechanism have made it possible to include the SPI bus choice as a parameter or parameters. Look at the `maxtherm` overlay to see how it's done.

Reply:
It's a pain to debug, but now it's working.

What's the best way to enable the SPI chip selects for spi1 and spi2? I've found this in the README:
```
        For devices on spi1 or spi2, the interfaces should be enabled
        with one of the spi1-1/2/3cs and/or spi2-1/2/3cs overlays.
```

Reply:
Do what maxtherm does, i.e. assume that spi1 and spi2 have been enabled and configured as necessary using one of the spi1-* and spi2-* overlays. All this overlay has to do is disabled any spidev instance on the CS it is using.

You can push what you've got so far if you'd like me to look it over.